src/store/index.js

```diff
@@ -58,18 +58,11 @@ export default createStore({
 				state.projects.push(project)
 			}
 		},
-
-		setImageCache(state, imageData) {
-			imageData.forEach(image => {
-				let imageArray
-				image.RLEVEL === 95 ? imageArray = state.smallImageCache : imageArray = state.largeImageCache
-				const existingImageIndex = imageArray.findIndex(img => img.basename === image.basename)
-				if (existingImageIndex === -1) {
-					imageArray.push(image)
-				} else {
-					imageArray[existingImageIndex] = image
-				}
-			})
+		setSmallImageCache(state, imageData){
```

Comment:
My humble opinion is that we should keep the setImageCache function as it was because:
1. it can be used for both large and small images
2. it checks to see if the image already exists in the array so we make less api requests

Reply:
I think a mutation should only mutate one state and any decision making between which mutation to call should be done in an action.
As for the logic in the old function it was actually doing extra work. At this point we've already made the api call and gotten the list of images so it doesn't make less api requests. The backend is the source of truth. So we don't even need to check for duplicates and just replace the entire array.

Reply:
@jnation3406 @mgdaily Carolina and I sat and talked about this and this is how she originally did it too, but someone said to put the logic for mutating both states into one mutation. Would like to hear your input on this if possible!